swagger.yaml

```diff
@@ -5479,6 +5480,19 @@ definitions:
       initrd_url:
         type: string
         description: URL address to download the initrd.
+      host_fs_mount_dir:
+        type: string
+        description: The base directory on the host that contains the /boot folder.
+
+  reboot_for_reclaim_request:
+    type: object
+    description: Information sent to the agent for rebooting a host into discovery.
+    required:
+      - host_fs_mount_dir
+    properties:
+      host_fs_mount_dir:
+        type: string
+        description: The base directory on the host that contains the /boot folder.
```

Comment:
>reboot_for_reclaim_request

How is the directory used in a `reboot_for_reclaim_request`? I would assume the agent would simply need to reboot at this stage, wouldn't it? Is it doing anything with that directory? If yes it could be nice if it were mentioned in the description

Reply:
Thanks for the feedback! I've modified the description to mention what it's used for